src/NServiceBus.Transport.Msmq.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -34,6 +34,27 @@ namespace NServiceBus
         public override NServiceBus.Transport.TransportInfrastructure Initialize(NServiceBus.Settings.SettingsHolder settings, string connectionString) { }
     }
 }
+namespace NServiceBus.Persistence.Legacy
+{
+    [System.ObsoleteAttribute("Classes under the NServiceBus.Persistence.Legacy namespace has been moved to NSer" +
```

Comment:
>Classes under the NServiceBus.Persistence.Legacy namespace has been moved to NServiceBus. 

This seems incorrect. 

Should we just make it simpler? 
Classes under the NServiceBus.Persistence.Legacy namespace has been moved. Use NServiceBus.MsmqPersistence instead. (etc..) ?

Reply:
isnt it sufficient to have these in the core repo? given this is a new v1 repo the obsolete doesnt really make sense

> Classes under the NServiceBus.Persistence.Legacy namespace has been moved to NServiceBus. Please use NServiceBus.MsmqPersistence instead

ie they were never in legacy in this package. i think obsolete in code and doco in upgrade guide should be sufficient

Reply:
>Classes under the NServiceBus.Persistence.Legacy namespace has been moved. Use NServiceBus.MsmqPersistence instead. (etc..) ?

This was more or less the original message I had. You wanted that to be changed @indualagarsamy ?

Reply:
I think @SimonCropp makes a good point here.

If we tighten up the obsoletes in the core to point to the actual replacement classes AND the replacement package instead of just the replacement package(as it is now) that is a better solution.

Since the classes have moved both to another package AND to another namespace, we should use the initial obsolete message to give all of that information. 

I'll create a PR for better obsolete messages in the core.